lib/rubocop/cop/mixin/enforce_superclass.rb

```diff
@@ -1,7 +1,9 @@
 # frozen_string_literal: true
 
 module RuboCop
-  module Cop
+  module Cop # rubocop:disable Style/Documentation
```

Comment:
Because this `RuboCop::Cop` module is no longer purely just a wrapper/namespace, since below we are adding `remove_const(:EnforceSuperclass) if defined?(EnforceSuperclass)` to it, RuboCop now expects a documentation comment for it. But I don't think that we really want to document this module here, so I am disabling the cop here.